clients/venice-admin-tool/src/main/java/com/linkedin/venice/KafkaTopicDumper.java

```diff
@@ -491,8 +491,8 @@ private void logRecordMetadata(DefaultPubSubMessage record) {
       final String chunkMetadata = getChunkMetadataLog(record);
 
       LOGGER.info(
-          "Offset:{}; {}; {}; ProducerMd=(guid:{},seg:{},seq:{},mts:{},lts:{}); LeaderMd=(host:{},uo:{},ukcId:{}){}",
-          record.getPosition().getNumericOffset(),
+          "Position:{}; {}; {}; ProducerMd=(guid:{},seg:{},seq:{},mts:{},lts:{}); LeaderMd=(host:{},uo:{},ukcId:{}){}",
```

Comment:
This will produce very verbose output, but we do need to remove getNumericOffset eventually. I was thinking we could build a relative index from the starting position and print that, while making the option to print the full position available as an extra feature. We can do that in a followup PR